mockautoconsumers.go

```diff
@@ -505,17 +505,17 @@ func (cg *MockConsumerGroup) topicKey(topics []string) string {
 	return strings.Join(topics, ",")
 }
 
-func (cg *MockConsumerGroup) markMessage(msg *sarama.ConsumerMessage) {
+func (cg *MockConsumerGroup) markMessage(topic string, partition int32, offset int64, metadata string) {
 	cg.mMessages.Lock()
 	defer cg.mMessages.Unlock()
 
-	cnt := cg.messages[msg.Offset]
+	cnt := cg.messages[offset-1]
```

Comment:
@ubunatic it's not the offset value that is being checked here, but whether a message with that offset exists. If it doesn't exist (cnt==0), then it panics, because the consumer is trying to "commit" a message with an offset, that it never received.